rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
```

Comment:
@jfmengels If I understood you properly, something like this
```js
/^(?:\\.|[\w ])+$/gu
```
Should do the trick to match a string with only "words" in it or escaped symbols.

See: https://regex101.com/r/3WKMiS/1/tests